### ISSUE TITLE
Resolve issues related to crashes during game loading.

### ISSUE THREAD

**Issue by p2004a** [beyond-all-reason/spring#329]
There is a known pattern of "please don't alt+tab or click anything when game loads or it might crash" that happening quite often for players, this is to track resolution of that.

I'm not including any logs/stacktraces in this issue at the moment, for now ensuring there is *something* open for tracking and we will add more details with reports from people in `#bugreports`/`#tech-support` channels from Discord etc.

**Comment by lhog** [comment]
Yes, good idea, especially given the fact that I removed unreliable `MTLoading` solution from another era. This will be an interesting software engineering challenge as OpenGL calls will need to be processed by a main thread, while non-GL calls can be offloaded to async tasks.
